scaffolds/streamlit_on_cloudrun/image_classification/train_and_deploy.ipynb

```diff
@@ -0,0 +1,785 @@
+{
```

Comment:
<p>I would push into github the following files that are generated by this notebook:</p><ul><li>app.py</li><li>cloudbuild.yaml</li><li>Dockerfile</li><li>requirements.txt</li></ul><p>That way, people browsing our repo for code snippet may be able to use that code without having to run the notebook. It also makes it clear the file structure of the app, and its logic. </p><p> </p><p>I'd also add a simple <code>deploy.sh</code> script collecting all the bash instructions in the notebook to deploy app (with a comment that the model needs to be trained and saved first in export). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>That makes sense, and adding <code>deploy.sh</code> sounds like a great idea!</p><p>Added all the file + deploy.sh, and included a README file for those instructions.</p><p>Thanks! </p><p> </p><p> </p>